faq.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta property="og:type" content="website" />
```

Comment:
```suggestion
<html lang="ko">
<head>
  <meta charset="UTF-8">
</head>
```

html lang="ko" 값으로 변경 되어야합니다. 😸

```
- 언어 하위태그(필수)
두 세 글자로 구성된 코드로 기본 언어를 정의하며, 보통 모두 소문자로 표기합니다. 예를 들어 영어의 언어 태그는 en이고, 한국어의 언어 태그는 ko입니다.
```

https://developer.mozilla.org/ko/docs/Web/HTML/Global_attributes/lang

Reply:
여기에 그리고 현재 탭 값이 일부 다른데 prettier가 정상으로 동작하는지 확인 필요할 듯 해요. 어디는 Tab이 4이고, 2이고 불규칙해서요. 보통은 tab width 값 2를 많이 사용하고는 합니다. (4를 선호하는 경우도 있습니다.)